sfge/src/main/java/com/salesforce/rules/RuleUtil.java

```diff
@@ -83,6 +83,59 @@ public static List<MethodVertex> getPathEntryPoints(
         return new ArrayList<>(methods);
     }
 
+    /**
+     * Indicates whether the provided method counts as a path entry point
+     * for the purposes of DFA execution.
+     */
+    public static boolean isPathEntryPoint(MethodVertex methodVertex) {
```

Comment:
Lets us identify which methods should be excluded.

Reply:
This feels like another portion that can be moved in with other entry point related logic.